test/cmd/images.sh

```diff
@@ -57,6 +57,9 @@ os::cmd::try_until_success 'oc get imagestreamtags wildfly:latest'
 os::cmd::expect_success_and_text "oc get imageStreams wildfly --template='{{ index .metadata.annotations \"openshift.io/image.dockerRepositoryCheck\"}}'" '[0-9]{4}\-[0-9]{2}\-[0-9]{2}' # expect a date like YYYY-MM-DD
 os::cmd::expect_success_and_text 'oc get istag' 'wildfly'
 
+# we have an istag here, let's see if the debug command works properly on it
+os::cmd::expect_success_and_text "oc debug istag/wildfly:latest -o yaml" 'image: openshift/wildfly-100-centos7'
```

Comment:
> Smells like we want test/cmd/debug.sh to be honest

We have one, but it doesn't import images.  Importing images is expensive, so this makes use a place that already imports them.